roles/lib_openshift/src/lib/rolebinding.py

```diff
@@ -0,0 +1,276 @@
+# pylint: skip-file
```

Comment:
There is not a standalone for rolebinding yet.  The modeling of the object exists but nothing for oc_rolebinding.